.changeset/honest-shirts-obey.md

```diff
@@ -0,0 +1,5 @@
+---
+"@vygruppen/spor-icon": minor
```

Comment:
When you add icons, you should also create a new release for `spor-icon-react` and `spor-icon-react-native`:

```suggestion
"@vygruppen/spor-icon": minor
"@vygruppen/spor-icon-react": minor
"@vygruppen/spor-icon-react-native": minor
```